src/hmrc-content-guide/extra-statutory-concessions/index.njk

```diff
@@ -6,7 +6,7 @@ layout: twoColumnPage.njk
 
 <h2 class="govuk-heading-l">Document type</h2>
 
-<p class="govuk-body">Guidance with HTML</p>
+<p class="govuk-body">Guidance (with a HTML attachment)</p>
```

Comment:
No idea about this but should HTML be preceded by 'an' rather than 'a'?

Reply:
Good spot, you're right. Just amended this across a few pages.